packages/components/src/components/Link/Link.test.tsx

```diff
@@ -0,0 +1,420 @@
+      type: 'directions',
+      locationData: {
+        latitude: location.lat,
+        longitude: location.long,
+        name: location.name,
+      },
+      text: 'Get Directions',
+    }
+    beforeEach(() => {
+      component = render(<Link {...directionsProps} />)
+    })
+
+    it('renders directions link', async () => {
+      const linkText = component.getByText('Get Directions')
+
+      expect(linkText).toBeDefined()
+    })
+
+    it('calls useExternalLink hook when tapped', async () => {
+      fireEvent.press(component.getByText('Get Directions'))
+
+      expect(useExternalLinkHookMock).toHaveBeenCalledWith('https://maps.apple.com/?t=m&daddr=%2BTibor+Rubin+VA+Medical+Center%2B33.7764681%2C-118.1189664', undefined, undefined)
+      expect(onPressSpy).not.toHaveBeenCalled()
+    })
+
+    it('renders directions icon', async () => {
+      const icon = component.root.findByType(Icon)
+
+      expect(icon).toBeDefined()
+      expect(icon.props.name).toBe('Directions')
+    })
+  })
+
+  describe('text variant tests', () => {
+    const textProps: LinkProps = {
+      type: 'text',
+      textNumber: '123456',
+      text: 'Text 123456',
+    }
+    beforeEach(() => {
+      component = render(<Link {...textProps} />)
+    })
+
+    it('renders text link', async () => {
+      const linkText = component.getByText('Text 123456')
+
+      expect(linkText).toBeDefined()
+    })
+
+    it('calls onPress when tapped', async () => {
+      fireEvent.press(component.getByText('Text 123456'))
+
+      expect(useExternalLinkHookMock).toHaveBeenCalledWith('sms:123456', undefined)
+      expect(onPressSpy).not.toHaveBeenCalled()
+    })
+
+    it('renders mobile phone icon', async () => {
+      const icon = component.root.findByType(Icon)
+
+      expect(icon).toBeDefined()
+      expect(icon.props.name).toBe('Text')
+    })
+  })
+
+  describe('url variant tests', () => {
+    const urlProps: LinkProps = {
+      type: 'url',
+      url: 'https://www.va.com',
+      text: 'External Link',
+    }
+    beforeEach(() => {
+      component = render(<Link {...urlProps} />)
+    })
+
+    it('renders url link', async () => {
+      const linkText = component.getByText('External Link')
+
+      expect(linkText).toBeDefined()
+    })
+
+    it('calls onPress when tapped', async () => {
+      fireEvent.press(component.getByText('External Link'))
+
+      expect(useExternalLinkHookMock).toHaveBeenCalledWith('https://www.va.com', undefined, undefined)
+      expect(onPressSpy).not.toHaveBeenCalled()
+    })
+
+    it('renders external link icon', async () => {
+      const icon = component.root.findByType(Icon)
+
+      expect(icon).toBeDefined()
+      expect(icon.props.name).toBe('ExternalLink')
+    })
+  })
+
+  describe('light mode tone tests', () => {
+    it('renders primary tone', async () => {
+      component = render(<Link {...defaultProps} />)
+      textColor = getTextColor(component)
+      expect(textColor).toBe('#005ea2')
```

Comment:
I used color tokens in my Button tests. I see the value in using the hex codes too, but I think we should stick to one of them for consistency. What do you think?

Reply:
Discussed in Slack and an AC was added to #237 to address this as the tokens are a bit in flux and short term we don't want to chase them with unit tests being brittle and breaking as the color tokens evolve.